packages/puppeteer-core/src/common/ChromeTargetManager.ts

```diff
@@ -100,15 +100,20 @@ export class ChromeTargetManager extends EventEmitter implements TargetManager {
 
   // TODO: remove the flag once the testing/rollout is done.
   #tabMode = false;
```

Comment:
Change to boolean, as we set it in the constructor that way we know where the value is coming from and that is not hard coded.